docker-storage-setup.sh

```diff
@@ -931,46 +938,81 @@ create_lv() {
 return 0
 }
 
-setup_docker_root_volume() {
-  if [ -z "$DOCKER_ROOT_VOLUME_SIZE" ]; then
-    Fatal "Specify a valid value for DOCKER_ROOT_VOLUME_SIZE."
+setup_extra_volume() {
+  local lv_name=$1
+  local mount_dir=$2
+  local lv_size=$3
+
+  if [ -z "$lv_size" ]; then
+    Fatal "Specify a valid value for CONTAINER_ROOT_LV_SIZE."
```

Comment:
This function is used by both docker and container volumes so we don't know if CONTAINER_ROOT_LV_SIZE is wrong or DOCKER_ROOT_LV_SIZE is wrong.

Can we move this check out of this function and do the check in the caller. So we will do this check in setup_docker_root_lv_fs() and in  setup_extra_lv_fs().